### PR TITLE
Improve documentation around @LocalstackDockerProperties

### LINKED ISSUE #1324: Docker example with junit5 throws java.lang.IllegalStateException
looks like this snippet for Junit5 (have not tested it with Junit4) is not working with `lambda` and `stepfunctions`

```
@ExtendWith(LocalstackDockerExtension.class)
@LocalstackDockerProperties(randomizePorts = true, services = { "sqs", "kinesis:77077" })
public class MyDockerCloudAppTest {
   ...
}
```

I get the error
```
java.lang.IllegalStateException: Service to port mapping has not been determined yet.

	at cloud.localstack.docker.LocalstackDocker.endpointForService(LocalstackDocker.java:183)
	at cloud.localstack.docker.LocalstackDocker.getEndpointLambda(LocalstackDocker.java:116)
	at cloud.localstack.DockerTestUtils.createEndpointConfiguration(DockerTestUtils.java:114)
	at cloud.localstack.DockerTestUtils.getClientLambda(DockerTestUtils.java:53)
```

Which seems misleading because even if I specifically set the port mapping localstack seems to ignore it. I also noticed that stepfunctions client is only available in the `DockerTestUtils` class and not in the `TestUtils` class so I cannot even try to do this without Docker.

### PR DESCRIPTION
**Pull request by ImFlog**
Fixes #1324.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -386,12 +386,25 @@ Or with JUnit 5 :
...
+| services                    | Determines which services should be run when the localstack starts.                                                          | String[]                     | All           |   |
+| imageTag                    | Use a specific image tag for docker container                                                                                | String                       | latest        |   |
+| hostNameResolver            | Used for determining the host name of the machine running the docker containers so that the containers can be addressed.     | IHostNameResolver            | localhost     |   |
+| environmentVariableProvider | Used for injecting environment variables into the container.                                                                 | IEnvironmentVariableProvider | Empty Map     |   |
+
+_NB : When specifying the the port in the `services` property, you cannot use `randomizePorts = true`_
+
 The *LocalStack* JUnit test runner is published as an artifact in Maven Central.
 Simply add the following dependency to your `pom.xml` file:
```
Comment (whummer) [inline README.md]: I'd move the new table below the following paragraph. Configuring the dependency in `pom.xml` seems like higher priority than the `@LocalstackDockerProperties` customizations.

File: README.md [hunk 2]
```diff
@@ -386,12 +386,25 @@ Or with JUnit 5 :
...
+| randomizePorts              | Determines if the container should expose the default local stack ports (4567-4583) or if it should expose randomized ports. | boolean                      | false         |   |
+| services                    | Determines which services should be run when the localstack starts.                                                          | String[]                     | All           |   |
+| imageTag                    | Use a specific image tag for docker container                                                                                | String                       | latest        |   |
+| hostNameResolver            | Used for determining the host name of the machine running the docker containers so that the containers can be addressed.     | IHostNameResolver            | localhost     |   |
+| environmentVariableProvider | Used for injecting environment variables into the container.                                                                 | IEnvironmentVariableProvider | Empty Map     |   |
+
+_NB : When specifying the the port in the `services` property, you cannot use `randomizePorts = true`_
```
Comment (whummer) [inline README.md]: nit: "the the"

File: README.md [hunk 3]
```diff
@@ -386,12 +386,25 @@ Or with JUnit 5 :
...
-@LocalstackDockerProperties(randomizePorts = true, services = { "sqs", "kinesis:77077" })
+@LocalstackDockerProperties(services = { "sqs", "kinesis:77077" })
 public class MyDockerCloudAppTest {
    ...
 }
 ```
 
+You can configure the Docker behaviour using the `@LocalstackDockerProperties` annotation with the following parameters:
+
+| property                    | usage                                                                                                                        | type                         | default value |   |
```
Comment (whummer) [inline README.md]: nit: seems like there is an unused additional column on the right side of the table which can be removed

**Comment by ImFlog** [comment]
Changes done according to your remarks @whummer

**Pull request merged**